lib/lita/handlers/reviewme.rb

```diff
@@ -96,16 +96,16 @@ def comment_on_github(response)
         end
       end
 
-      def mention_reviewer(response)
+      def mention_reviewer(response, room: get_room(response))
         url = response.matches.flatten.first
-        reviewer = next_reviewer
+        reviewer = next_reviewer(room)
         response.reply(chat_mention(reviewer, url))
       end
 
       private
 
-      def next_reviewer
-        redis.rpoplpush(REDIS_LIST, REDIS_LIST)
+      def next_reviewer(room)
+        ns_redis(room).rpoplpush(REDIS_LIST, REDIS_LIST)
```

Comment:
So I found out that all redis connections within a plugin are already namespaced. So `redis` in this case would be something like `lita:reviewme` as the key. This just adds another bit to that so you end up with something like `lita:reviewme:R12345` (slack channel id). Does that make sense?